docs/development/remote.md

```diff
@@ -37,14 +37,13 @@ to the next section.
 
 You can create a new user with sudo privileges by running the
 following commands as root:
-- You can create a `zulipdev` user by running the command `adduser
-zulipdev`. Run through the prompts to assign a password and user
-information. (You can pick any username you like for this user
-account.)
+
+- You can create a `zulipdev` user by running the command `adduser zulipdev`. Run through the prompts to assign a password and user
```

Comment:
Another example of surprising line-wrapping, I assume caused by us previously having the inline code block split across 2 lines.